packages/api/db/migration/20210805200615_products.js

```diff
@@ -18,7 +18,7 @@ exports.up = async function(knex) {
   await knex.raw('ALTER TABLE fertilizer_task DROP CONSTRAINT fertilizerlog_fertilizer_id_foreign');
   await knex.schema.renameTable('fertilizer_task', 'soil_amendment_task');
   await knex.schema.alterTable('soil_amendment_task', (t) => {
-    t.dropColumn('fertilizer_id');
+    t.renameColumn('fertilizer_id', 'historic_product_id');
```

Comment:
is changing an existing migration that has already been run considered not good?